lib/text.rb

```diff
@@ -0,0 +1,28 @@
+require 'twilio-ruby'
+
+class Text
+
+  attr_reader :message
+
+  def send(working_number = "+441202835326", my_account_sid = ENV["TWILIO_ACCOUNT_SID"], my_auth_token = ENV["TWILIO_AUTH_TOKEN"])
```

Comment:
Metrics/LineLength: Line is too long. [130/80] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)